### DISCUSSION TITLE
Migration of the Cruiser setup inside my new smartphone.

### DISCUSSION THREAD

**Discussion by jclaudej** [devemux86/cruiser#347] [Q&A] [answered]
Hi devemux86

I would like to know if it's possible to copy the Cruiser setup to another smartphone Android ?

There are many setup possibilities for Cruiser and it would be comfortable to be able to migrate easily by recovering its settings

Thank a lot for your help ;)

**Comment by devemux86** [comment, marked as answer]
Hi @jclaudej,

You can transfer easily the app settings, they are stored in your active app profile:
- Press the app menu button, press the profile at the top, press the active profile's menu and select "Export"
- Transfer the profile file to the other device and follow the same process selecting "Import" (as new profile or update)

A similar process can be followed for the favorites,
and in the [next Cruiser](https://github.com/devemux86/cruiser/discussions/329) for the new favorite routes + tracks.

Please let me know if you need more help!

**Comment by jclaudej** [comment]
Thank a lot for your help.
It’s easy with your explanation. ✌️😉

Have nice day

Jean-Claude

Envoyé de mon iPad

Le 17 oct. 2023 à 12:42, Emux ***@***.***> a écrit :


You can transfer easily the app settings, they are stored in your active app profile:

  *   Press the app menu button, press the profile at the top, press the active profile's menu and select "Export"
  *   Transfer the profile file to the other device and follow the same process selecting "Import" (as new profile or update)

A similar process can be followed for the favorites,
and in the next Cruiser<https://github.com/devemux86/cruiser/discussions/329> for the new favorite routes + tracks.

Please let me know if you need more help!
